Experiments_toolkit.py

```diff
@@ -57,151 +50,143 @@ def splitting_up_data(dictionary, dict_key, train, percent_train, test, percent_
+       Description: Preprocess the data stored in data by windowing the traces and saving
                                 these windows (numpy arrays) into a dictionary.
 
        Args:
-                DATA -- np array (Numpy array where the last row is time steps)
+            data -- list (List of numpy arrays where the last row is time steps, 
+            			  [(sensor_dims + 1), num_samples])
             rows -- int (Number of rows in the data)
             filtering_params -- All_Filtering_Params (Dataclass found in windowing.py that 
-                                                        stores important parameters for the 2 steps involved with 
-                                                        finding the precise start and end time of a lift in the data)
-            window_size -- int (Window size of the trace to be saved into the dictionary that will be
-                                                returned)
-            offset -- int (Number to add to the predicted start time of the lift from where the window
-                                                will start at)
+                                stores important parameters for the 2 steps involved with 
+                                finding the precise start and end time of a activity in the data)
+            window_size -- int (Window size of the trace to be saved into the
+            				    dictionary that will be returned)
+            offset -- int (Number to add to the predicted start time of the 
+            			   activity from where the window will start at)
 
        Return:
-           windowed_data_dictionary -- dict (Dictionary that stores the windows under the "windows" key)
+           windows-- int[] (Array of the windows cropped by the window_function)
            """
-    windowed_data_dictionary = {"windows": []}
-    for j in range(len(DATA)):
-        sample = DATA[j]
-        lift_windows = wl.initial_find_lift(
-            sample, rows, filtering_params.initial_filtering.window_size, filtering_params.initial_filtering.stride, filtering_params.initial_filtering.divisor)
-        if len(lift_windows) > 0:
+    windows = []
+    for j in range(len(data)):
+        sample = data[j]
+        activity_windows = wl.initial_find_activity(
+            sample, rows, filtering_params.initial_filtering.window_size,
+            filtering_params.initial_filtering.stride,
+            filtering_params.initial_filtering.divisor)
+        if len(activity_windows) > 0:
             start_time, end_time = wl.get_start_time(
-                lift_windows, rows, filtering_params.precise_filtering.window_size, filtering_params.precise_filtering.stride, filtering_params.precise_filtering.threshold)
+                activity_windows, rows, filtering_params.precise_filtering.window_size,
+                filtering_params.precise_filtering.stride,
+                filtering_params.precise_filtering.threshold)
             pos_window = window_function(
```

Comment:
nit: since you have functions that work with positive/negative data, pos may be misleading here